src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -97,7 +99,7 @@ export class SkyAutonumericDirective implements OnInit, ControlValueAccessor, Va
       }
     }
 
-    if (!Number.isNaN(value) && value !== undefined && value !== null) {
+    if (typeof value === 'number' && value !== null && value !== undefined) {
```

Comment:
☝️ `Number.isNaN` isn't supported by Internet Explorer.

Reply:
Could you use the global `isNaN` function instead?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN